src/components/PrivateRoute.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { Route, Redirect } from "react-router-dom";
+import { connect } from "react-redux";
+
+const PrivateRoute = ({ component: Component, ...rest }) => {
+  return (
+    <Route
+      {...rest}
+      render={(props) =>
+        rest.isAuth ? <Component {...props} /> : <Redirect to="/" />
```

Comment:
Should it redirect to dash or login?

Reply:
I guess it's up to the design decision do we want the user to see the dashboard when they hit the private route or do we want them instead to login?